### PR TITLE
Av/42 add icons

### PR DESCRIPTION
**Pull request by anushkavidanage**
# Pull Request Template

## What are you trying to address

- Adding app default icons for all platforms

## Checklist

[READY TO PR? Use the check-list below to ensure your branch is ready for PR.]

- [x] Changes follow the repo structure and land in the appropriate folder and section
- [x] No confidential information
- [x] No duplicated content
- [x] Labeled appropriately
- [x] Added 2 reviewers
- [x] No lint errors
- [x] No lint check errors related to your changes

### CODE REVIEW

**Review by zheyxu — APPROVED** [review]
The GitHub action fails because ```Because keypod depends on solid from path which doesn't exist (could not find package solid at "../solid"), version solving failed.```

It need further discussion as we only keep solid project locally.

**Comment by gjwgit** [comment]
Is the change to ../solid the problem? In GitHUb actions can't we clone the solid package into 'solid'. The '../solid' might be a problem in that context?

File: pubspec.yaml
Comment (gjwgit) [inline pubspec.yaml:1]: Is there a need for it to be `../solid` for the path. The GitHub actions may rely on it being `solid`, for example.

**Review by gjwgit — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 1]
```diff
@@ -17,13 +17,36 @@ dependencies:
...
-    ##   ref: main
+    path: ../solid # Use a local path whilst solid is under development.
+    # git:
+    #   url: https://github.com/anusii/solid
+    #   ref: main
   universal_io: ^2.2.2
   window_manager: ^0.3.7
+  
+
+# Define laucher icons for all platforms
```
Comment (gjwgit) [inline pubspec.yaml:30]: `laucher` needs an `n` and please add a blank line after the comment. Thanks.

File: pubspec.yaml
Comment (anushkavidanage) [reply pubspec.yaml:1]: This actually depends on where exactly we keep our local copy of the solid package. For me it needs to be `../solid` in order for the app to work

File: pubspec.yaml (hunk 1 above)
Comment (anushkavidanage) [reply pubspec.yaml]: Resolved

File: pubspec.yaml
Comment (NinadBhat) [reply pubspec.yaml:1]: Hi, @anushkavidanage. The GitHub workflow finds and deletes the line `path: solid`. After this, it points the package link to the URL "https://github.com/anusii/solid". Hence, we can't change the path for the online repo. You can change it locally if you'd like to.

Comment (gjwgit) [reply pubspec.yaml:1]: Hi @anushkavidanage Could you revert this section of `pubspec.yaml` to the original then it can be approved and merged. Thanks.

**Review by gjwgit — CHANGES_REQUESTED** [review]

**Comment by anushkavidanage** [comment]
Reverted back to the original path

**Review by gjwgit — APPROVED** [review]

**Comment by gjwgit** [comment]
This can be merged now. Thanks @anushkavidanage.

**Pull request merged**